packages/dashboards/app/styles/navi-dashboards/common.less

```diff
@@ -43,16 +43,29 @@
     margin-right: 10px;
 
     .action {
-      background: transparent;
       color: @body-font-gray;
       font-size: @font-size-base-large;
 
+      &:hover {
+        background-color: @navi-gray-200;
+      }
+
       .icon {
         font-size: @font-size-base-large;
         position: relative;
         top: -1px;
       }
 
+      &.delete button {
+        font-weight: 400;
```

Comment:
maybe use `btn-secondary` instead

Reply:
No, the problem here is that `btn` and `btn-secondary` are applied to different elements and thats causing an issue. These were just overrides so that this button looks the same as the other components without having to change all of the styles.